.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+**Description**: Describe your changes.
+
+**Related issue:** Add link to related issue.
+
+**Documentation changes required:** (Y/N) Mark if documentation changes are required.
+
+**Added tests:** (Y/N) Mark if tests were added or updated according changes.
```

Comment:
I'd re-write to:
`**Test updates** - [ ] Mark if tests were added or updated according to the changes.`

Reply:
Fixed.